robot-command/src/main/java/org/obolibrary/robot/DiffCommand.java

```diff
@@ -47,7 +47,12 @@ public DiffCommand() {
     o.addOption("r", "right", true, "load right ontology from file");
     o.addOption("R", "right-iri", true, "load right ontology from IRI");
     o.addOption("o", "output", true, "save results to file");
-    o.addOption(null, "labels", true, "if true, use labels in place of entity IRIs");
+    o.addOption(
+        null,
+        "labels",
+        true,
+        "if true, append labels after entity IRIs in the simple format output");
+    o.addOption("f", "format", true, "format for diff output: simple (default) | html | markdown");
```

Comment:
The other formats are actual formats (HTML or MD) whereas simple is just plain-text. Maybe instead of `simple`, it could be `txt` or `text` for plain-text?

Reply:
I agree, I will update.

Reply:
Addressed in 3048bb8935c9a65d56480abccf7ab3d7d648100c.